cdc/log.cc

```diff
@@ -68,6 +70,62 @@ namespace cdc {
 static schema_ptr create_log_schema(const schema&, std::optional<utils::UUID> = {});
 }
 
+static constexpr auto cdc_group_name = "cdc";
+
+void cdc::stats::parts_touched_stats::register_metrics(seastar::metrics::metric_groups& metrics, sstring suffix) {
```

Comment:
Nit: suffix can be taken by const sstring& or string_view, no?

Reply:
Changed to string_view